widgets/common/src/filters/common.ts

```diff
@@ -6,7 +6,11 @@ const i18nPrefix = 'widgets.common.filters.form';
 export const getTranslation = (key: string) => i18n.t(`${i18nPrefix}.${key}`);
 export const getPlaceholderTranslation = (key: string) => getTranslation(`inputsPlaceholders.${key}`);
 
-const anyOperators: FilterRuleOperator[] = [FilterRuleOperators.AnyOf, FilterRuleOperators.NotAnyOf];
-export function isAnyOperator(operator: FilterRuleOperator) {
-    return anyOperators.includes(operator);
+const multipleValuesOperators: FilterRuleOperator[] = [
+    FilterRuleOperators.AnyOf,
+    FilterRuleOperators.NotAnyOf,
+    FilterRuleOperators.IsNot
+];
+export function isMultipleValueOperator(operator: FilterRuleOperator) {
```

Comment:
I like the new name :+1: Optionally, I would name it

```suggestion
export function isMultipleValuesOperator(operator: FilterRuleOperator) {
```

Reply:
Make sense. Changed.